addon/templates/components/hero.hbs

```diff
@@ -12,7 +13,7 @@
     )}}
   </div>
 
-  {{#if (or source credit caption)}}
+  {{#if (and (not noHeaderImage) (or source credit caption))}}
```

Comment:
This logic is a little tough to read. I'd like to see it as a computed property on the component, which you can do with something like:
```javascript
showCredits: computed('noHeaderImage', 'source', 'credit', 'caption', function() {
  let { noHeaderImage, source, credit, caption } = this.getProperties('noHeaderImage', 'source', 'credit', 'caption');
 return !noHeaderImage && (source || credit || caption);
})
```